superset/views/core.py

```diff
@@ -1853,11 +1853,12 @@ def dashboard(  # pylint: disable=too-many-locals
         }
 
         return self.render_template(
-            "superset/dashboard.html",
-            entry="dashboard",
+            "superset/crud_views.html",
```

Comment:
We should, eventually, rename this template to `spa.html`

Reply:
seems like now could be the time